edk2toolext/environment/plugin_manager.py

```diff
@@ -82,32 +82,34 @@ def _load(self, PluginDescriptor):
             PluginDescriptor(PluginDescriptor): the plugin descriptor
         """
         PluginDescriptor.Obj = None
-        PythonFileName = PluginDescriptor.descriptor["module"] + ".py"
-        PyModulePath = os.path.join(os.path.dirname(os.path.abspath(
-            PluginDescriptor.descriptor["descriptor_file"])), PythonFileName)
-        logging.debug("Loading Plugin from %s", PyModulePath)
-        try:
-            with open(PyModulePath, "r") as plugin_file:
-                _module = imp.load_module(
-                    "UefiBuild_Plugin_" + PluginDescriptor.descriptor["module"],
-                    plugin_file,
-                    PyModulePath,
-                    ("py", "r", imp.PY_SOURCE))
 
+        py_file_path = PluginDescriptor.descriptor["module"] + ".py"
+        py_module_path = os.path.join(os.path.dirname(os.path.abspath(
+            PluginDescriptor.descriptor["descriptor_file"])), py_file_path)
+        py_module_name = "UefiBuild_Plugin_" + PluginDescriptor.descriptor["module"]
+
+        logging.debug("Loading Plugin from %s", py_module_path)
```

Comment:
`logging.debug(f'Loading Plugin from {py_module_path})`

I find this way of doing variable insertion into strings much more readable. Not forcing you to make the change though.

Reply:
I do too but that comment is just left as-is from before (apart from the variable name refactor). With many of these refactor changes, there's eventually a line as to whether it is better to fix isolated cases like this or go through the whole codebase and use f-strings.

I decided not to cross the line in this case since this is not a new `logging.debug` addition to the codebase and the previous style is not at risk of being deprecated/removed (at least to my knowledge).

Reply:
Fair enough. I’ll raise an issue to go through and replace them all.